src/server/service/draft_service.py

```diff
@@ -0,0 +1,38 @@
+from src.server.ygo_card_db_service import YGOCardDBService
+
+class InvalidCreationError(Exception):
```

Comment:
Can you make an exceptions.py file and add these exceptions? I'd like to keep all the exceptions in a single file for ease of modifications.